src/caremote.c

```diff
@@ -197,6 +201,15 @@ CaRemote* ca_remote_unref(CaRemote *rr) {
         return mfree(rr);
 }
 
+int ca_remote_set_rate_limit_bps(CaRemote *rr, uint64_t rate_limit_bps) {
+        if (!rr)
+                return -EINVAL;
+
```

Comment:
an "if (rate_limit_bps == 0) return -EINVAL" might be nice... and similar on ca_sync_set_rate_limit_bps